#kia
vehicle_internal_state_topic: "/imu/velocity"
gps_topic: "/sbg/gps_pos" #"/ublox_gps/fix"
imu_topic: "/sbg/imu_data"
publish_topic: "VehStEstimatorDataNew"

#carla
# vehicle_internal_state_topic: "/imu/data"
# gps_topic: "/fix"
# imu_topic: "/odom"
# publish_topic: "VehStEstimatorDataNew"
